pycroft/lib/membership.py

```diff
@@ -120,4 +130,21 @@ def remove_member_of(user, group, processor, during=UnboundedInterval):
     message = deferred_gettext(u"Removed from group {group} during {during}.")
     log_user_event(message=message.format(group=group.name,
                                           during=during).to_json(),
-                   user=user, author=processor)
\ No newline at end of file
+                   user=user, author=processor)
+
+
+@with_transaction
+def edit_property_group(group, name, permission_level, processor):
+    log_event("Edited property group {} -> {}.".format((group.name, group.permission_level),
+                                                      (name, permission_level)),
+              processor)
+
+    group.name = name
+    group.permission_level = permission_level
+
+@with_transaction
+def delete_property_group(group, processor):
+    log_event("Deleted property group '{}'.".format(group.name),
+              processor)
+
```

Comment:
Shouldn't there be a permission_level check here?
`processor.permission_level  >= group.permission_level`